api/src/test/java/gov/cms/ab2d/api/controller/BulkDataAccessAPIIntegrationTests.java

```diff
@@ -242,4 +244,71 @@ public void testGetStatusWithBadUrl() throws Exception {
         this.mockMvc.perform(get("http://localhost" + API_PREFIX  + "/Job/$status").contentType(MediaType.APPLICATION_JSON))
                 .andExpect(status().is(404));
     }
+
+    @Test
+    public void testDownloadFile() throws Exception {
+        MvcResult mvcResult = this.mockMvc.perform(get(API_PREFIX + PATIENT_EXPORT_PATH + "?_type=ExplanationOfBenefits").contentType(MediaType.APPLICATION_JSON))
+                .andReturn();
+        String statusUrl = mvcResult.getResponse().getHeader("Content-Location");
+
+        Job job = jobRepository.findAll(Sort.by(Sort.Direction.DESC, "id")).iterator().next();
+        job.setStatus(JobStatus.SUCCESSFUL);
+        job.setProgress(100);
+        LocalDateTime expireDate = LocalDateTime.now().plusDays(100);
+        job.setExpires(expireDate);
+        LocalDateTime now = LocalDateTime.now();
+        job.setCompletedAt(now);
+
+        JobOutput jobOutput = new JobOutput();
+        jobOutput.setFhirResourceType("ExplanationOfBenefits");
+        jobOutput.setJob(job);
+        jobOutput.setFilePath("http://localhost" + API_PREFIX + "/Job/" + job.getJobID() + "/file/test.ndjson");
```

Comment:
JobOutput will never contain a full URL by the way, it should only contain a relative path to the file (relative to the EFS mount). The controller is responsible for constructing the URL itself. So in this particular example the file name should be just `test.ndjson`.